terraform/modules/spanner/main.tf

```
@@ -21,6 +21,7 @@ resource "google_spanner_instance" "main" {
   config       = "regional-${var.region}"
   display_name = var.spanner_name
   project      = var.project_id
+  num_nodes    = 1
```

```suggestion
  processing_units = 100
```
This change provisions a smaller instance by default.